pyproject.toml

```
@@ -43,9 +43,9 @@ tests = [
 ]
 
 [project.urls]
-homepage = "https://mne.tools/mne-bids-pipeline"
-repository = "https://github.com/mne-tools/mne-bids-pipeline"
-changelog = "http://github.com/mne-tools/mne-qt-browser/releases"
+homepage = "https://github.com/mne-tools/mne-qt-browser"
```

```suggestion
homepage = "https://mne.tools/"
```

let's just point to MNE.